Makefile

```diff
@@ -88,16 +73,16 @@ clean: clean-certs clean-gens clean-crates ## Clean the repo
 lint: auraed-lint not-auraed-lint ## Run all lints
 
 .PHONY: test
-test: auraed-build auraed-lint auraed-test not-auraed-build not-auraed-lint not-auraed-test ## Builds, lints, and tests (does not include ignored tests)
+test: os-deps auraed-build auraed-lint auraed-test not-auraed-build not-auraed-lint not-auraed-test ## Builds, lints, and tests (does not include ignored tests)
```

Comment:
probably not the right place to put it.  we want it where we check for buf so if but it's missing we can install it (same for the proto gen stuff) 

remove this and just keep this pr focused on the workflow changes to call it and I can clean all the dependency stuff up later.

Reply:
heard